emails/models.py

```diff
@@ -415,6 +415,11 @@ def is_flagged(self):
         return True
 
 
+def get_storing_phone_log(relay_number):
+    profile = Profile.objects.get(user=relay_number.user)
+    return profile.store_phone_log
```

Comment:
Nit: change to a one-liner?:

```suggestion
    return relay_number.user.profile_set.get().store_phone_log
```